features/bevragen/zoek-met-straatnaam-huisnummer-en-gemeente-van-inschrijving/fout-cases.feature

```diff
@@ -0,0 +1,234 @@
+Rule: Straat, huisnummer en gemeenteVanInschrijving zijn verplichte parameters
+
+  @fout-case
+  Scenario: De straat, huisnummer en gemeenteVanInschrijving parameters zijn niet opgegeven
+    Als personen wordt gezocht met de volgende parameters
+    | naam   | waarde                                           |
+    | type   | ZoekMetStraatHuisnummerEnGemeenteVanInschrijving |
+    | fields | burgerservicenummer                              |
+    Dan heeft de response een object met de volgende gegevens
+    | naam     | waarde                                                                      |
+    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1                 |
+    | title    | Minimale combinatie van parameters moet worden opgegeven.                   |
+    | status   | 400                                                                         |
+    | detail   | De foutieve parameter(s) zijn: gemeenteVanInschrijving, huisnummer, straat. |
+    | code     | paramsCombination                                                           |
+    | instance | /haalcentraal/api/brp/personen                                              |
+    En heeft het object de volgende 'invalidParams' gegevens
+    | code     | name                    | reason                  |
+    | required | straat                  | Parameter is verplicht. |
+    | required | huisnummer              | Parameter is verplicht. |
+    | required | gemeenteVanInschrijving | Parameter is verplicht. |
+
+  @fout-case
+  Scenario: Een lege string is opgegeven als straat, huisnummer en gemeenteVanInschrijving waarde
+    Als personen wordt gezocht met de volgende parameters
+    | naam                    | waarde                                           |
+    | type                    | ZoekMetStraatHuisnummerEnGemeenteVanInschrijving |
+    | fields                  | burgerservicenummer                              |
+    | straat                  |                                                  |
+    | huisnummer              |                                                  |
+    | gemeenteVanInschrijving |                                                  |
+    Dan heeft de response een object met de volgende gegevens
+    | naam     | waarde                                                                      |
+    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1                 |
+    | title    | Minimale combinatie van parameters moet worden opgegeven.                   |
+    | status   | 400                                                                         |
+    | detail   | De foutieve parameter(s) zijn: gemeenteVanInschrijving, huisnummer, straat. |
+    | code     | paramsCombination                                                           |
+    | instance | /haalcentraal/api/brp/personen                                              |
+    En heeft het object de volgende 'invalidParams' gegevens
```

Comment:
Zie de opmerkingen die ik hierover heb gemaakt in de fout-cases.feature bestanden bij de andere zoek types.